.circleci/config.yml

```diff
@@ -212,6 +220,16 @@ workflows:
                 - env/dev2
             tags:
               only: /^v.*$/
+      - deploy_api_docs:
+          requires: ~
```

Comment:
What does this line mean?

Reply:
Spec is [here](https://circleci.com/docs/2.0/configuration-reference/#requires), no mention of `~`
Let me see YAML spec

Reply:
I see it also [here](https://github.com/aeternity/epoch/blob/66ea4da55bb919eac10985c53ac920b46fba6cc3/.circleci/config.yml#L246) though not for linux_package

Reply:
@lucafavatella It means "null" AFAIK, I think the "requires" is required key
 still this line is obsolete and copy-paste leftover probably

Reply:
@michalbagrowski ^^